en/docs/tutorials/connect-your-existing-ballerina-project-to-choreo.md

```diff
@@ -41,6 +41,33 @@ Read and write access to code, pull requests, and repository hooks
 10. Review and click **Merge pull request**, and then click **Confirm Merge**. 
    When you merge the pull request, it adds the necessary metadata files to connect your Ballerina project repository to Choreo so that you can proceed to create the component.
 
+11. To configure the pre-commit hook that Choreo uses to extract configurables defined in the Ballerina code, follow these sub-steps:
```

Comment:
Do we need to say sub steps? 
Shouldn't they just be `steps given below`

Reply:
Changed.